packages/framework/src/Resources/views/Common/Inline/Icon/icon.html.twig

```diff
@@ -0,0 +1,8 @@
+{% if type == 'svg-font' %}
+    <i class="svg svg-{{ name }} {{ attr.class}}"
+        {% if attr.title is not same as('') %}title ="{{ attr.title }}"{% endif %}
+        {% for key, value in attr.data %}
+            data-{{ key }}="{{ value }}"
```

Comment:
If we want to be so consistent with symfony forms we should remove the `data-` prefix out of here and let it to be set via function. What do you think?

Reply:
Sorry I do not follow, can you elaborate on that?